testing/integration/upgrade_broken_package_test.go

```diff
@@ -25,6 +25,7 @@ import (
 
 func TestUpgradeBrokenPackageVersion(t *testing.T) {
 	define.Require(t, define.Requirements{
+		Group: "upgrade-specific",
```

Comment:
This is the only test group name that isn't self explanatory to me.

Maybe replace `upgrade-specific` with `upgrade-versions` since both this and `TestStandaloneDowngradeToSpecificSnapshotBuild` are about testing version related behavior?

Reply:
Okay.